### DISCUSSION TITLE
v1.3.0 Released

### DISCUSSION THREAD

**Discussion by cmacrae** [cmacrae/spacebar#71] [News]
This release is packed full of fixes and features ✨ Enjoy!  

## [1.3.0](https://github.com/cmacrae/spacebar/releases/tag/v1.3.0) - 2020-04-25
**Changed**
- Fixed a bug introduced in Big Sur where buffer reads were incorrect
- Improved efficiency of bar's initialisation
- Fixed DoNotDisturb indicator on Big Sur

**Added**
- New `left|center|right` shell sections: display custom text based on shell pipelines
- Option to turn focused window title display on or off (thanks [@Norviah](https://github.com/Norviah)!)
- Option to turn the spaces indicator on or off
- Option to turn the clock on or off
- Option to turn the power indicator on or off
- Option to turn the DoNotDisturb indicator on or off
- Option to set the padding between the first/last item and the left/right edge of the display
- Default to "Solid" style of Font Awesome 5 Free icon font
- Option to display spaces for all displays, including: optional separator, secondary & tertiary space indicator colours in relation to display
- Option to specify only drawing one bar on the main display or multiple bars, one for each display (this aligns with yabai's `external_bar` option)
- Provide a [Nix Flake](https://nixos.wiki/wiki/Flakes)

### Examples

**New space indicator for all displays**
![image](https://user-images.githubusercontent.com/3392199/116000897-f2899d00-a5e9-11eb-8e01-0faa4697bf65.png)

**Shell plugins from [the discussions thread](https://github.com/cmacrae/spacebar/discussions/70)**

Unread e-mail on the right  
![image](https://user-images.githubusercontent.com/3392199/116000981-48f6db80-a5ea-11eb-8462-1d49347c6143.png)

Current Kubernetes context on the right  
![image](https://user-images.githubusercontent.com/3392199/116000988-501de980-a5ea-11eb-8e31-ddc3911a46d1.png)

Currently playing artist/album in the centre
![image](https://user-images.githubusercontent.com/3392199/116000994-5744f780-a5ea-11eb-8278-343c1e0014ba.png)

CPU usage on the right  
![image](https://user-images.githubusercontent.com/3392199/116001007-5e6c0580-a5ea-11eb-9e84-30496044bb5a.png)

**Comment by ericlovesmath** [comment]
Very excited for this update, thank you for the work that you all do!

**Reply by cmacrae** [reply]
Glad you're happy with it! Thank you for the kind words 🙇
